### DISCUSSION TITLE
get always a blank page after login

### DISCUSSION THREAD

**Discussion by hapestr** [poweradmin/poweradmin#505] [Q&A] [answered]
Hi all,
i have installed a new powerdns server with poweradmin (3.5.0) as domain-manage-backend. If i login i always get a blank page, and have to load the complete url once more to get the cards overview.

I have the following setup

debian11
apache2.4
powerdns4.4.1 with mysql-backend
php8.1

Does anyone have an idea, what i'm doing wrong?

Regards

Hans-Peter Straub

**Comment by edmondas** [comment]
@hapestr Are there any errors in the http server logs? 

You can enable error display by setting `display_errors = on` in your php.ini file or by adding this code block to index.php.
```
ini_set('display_errors', 1);
ini_set('display_startup_errors', 1);
error_reporting(E_ALL);
```

**Reply by hapestr** [reply]
I get the following error when activating error_reporting:

Fatal error: Uncaught TypeError: openlog(): Argument #3 ($facility) must be of type int, string given in /var/www/html/poweradmin/lib/Logger.php:32 Stack trace: #0 /var/www/html/poweradmin/lib/Logger.php(32): openlog() #1 /var/www/html/poweradmin/lib/Logger.php(59): Poweradmin\Logger::do_log() #2 /var/www/html/poweradmin/lib/UserEventLogger.php(28): Poweradmin\Logger::log_notice() #3 /var/www/html/poweradmin/inc/plugins/auth_local/auth_local.plugin.php(214): Poweradmin\UserEventLogger::log_successful_auth() #4 /var/www/html/poweradmin/inc/plugins/auth_local/auth_local.plugin.php(79): SQLAuthenticate() #5 /var/www/html/poweradmin/inc/plugin.inc.php(81): authenticate_local() #6 /var/www/html/poweradmin/inc/toolkit.inc.php(69): do_hook() #7 /var/www/html/poweradmin/index.php(40): require_once('...') #8 {main} thrown in /var/www/html/poweradmin/lib/Logger.php on line 32

Should i deactivate logging?

**Reply by edmondas** [reply]
Thank you for providing the error trace. Can you tell me what value there is for $syslog_facility?
It should be an unquoted string taken from the following list of available facility constants:
https://www.php.net/manual/en/function.openlog.php

**Reply by hapestr** [reply, marked as answer]
Ok that was it. In my old config.inc.php taken from an old installation, there was an entry 

$syslog_facility = 'LOG_USER'

Removing the quotes and the login is working.

Thanks a lot for the quick help !!!!
